### ISSUE TITLE
ActionController::Parameters instances expect only one argument for :require

### ISSUE THREAD

**Issue by kiliancs** [rails/rails#27658]
The [documentation](http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require) for the method explains that _when given an array of keys, the method tries to require each one of them in order. If it succeeds, an array with the respective return values is returned_. Then it provides this example:

```
params = ActionController::Parameters.new(user: { ... }, profile: { ... })
user_params, profile_params = params.require(:user, :profile)
```

However, running the above (with the obvious changes) it raises an error.

### Steps to reproduce
Run the following in the rails console.
```
> params = ActionController::Parameters.new(user: { example: 1 }, profile: { example: 1 })
=> <ActionController::Parameters {"user"=>{"example"=>1}, "profile"=>{"example"=>1}} permitted: false>
> params.require(:user, :profile)
ArgumentError: wrong number of arguments (given 2, expected 1)
...
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.1/lib/action_controller/metal/strong_parameters.rb:304:in `require'
	from (irb):66
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/console.rb:65:in `start'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/console_helper.rb:9:in `start'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/commands_tasks.rb:78:in `console'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands.rb:18:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `block in require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /[...]/bin/rails:9:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `block in load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/commands/rails.rb:6:in `call'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/command_wrapper.rb:38:in `call'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:191:in `block in serve'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:161:in `fork'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:161:in `serve'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:131:in `block in run'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:125:in `loop'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:125:in `run'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

### Expected behavior
According to the [documentation](http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require) for the method _when given an array of keys, the method tries to require each one of them in order. If it succeeds, an array with the respective return values is returned_. The following example is provided:

```
params = ActionController::Parameters.new(user: { ... }, profile: { ... })
user_params, profile_params = params.require(:user, :profile)
```

### Actual behavior
```
ArgumentError: wrong number of arguments (given 2, expected 1)
from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/actionpack-5.0.1/lib/action_controller/metal/strong_parameters.rb:304:in `require'
	from (irb):66
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/console.rb:65:in `start'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/console_helper.rb:9:in `start'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/commands_tasks.rb:78:in `console'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands/commands_tasks.rb:49:in `run_command!'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/railties-5.0.1/lib/rails/commands.rb:18:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/backports-3.6.8/lib/backports/std_lib.rb:9:in `require_with_backports'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `block in require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:293:in `require'
	from /[...]/Documents/huddle/bin/rails:9:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `block in load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:259:in `load_dependency'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/activesupport-5.0.1/lib/active_support/dependencies.rb:287:in `load'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/commands/rails.rb:6:in `call'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/command_wrapper.rb:38:in `call'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:191:in `block in serve'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:161:in `fork'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:161:in `serve'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:131:in `block in run'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:125:in `loop'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application.rb:125:in `run'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/spring-2.0.0/lib/spring/application/boot.rb:19:in `<top (required)>'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /[...]/.rbenv/versions/2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from -e:1:in `<main>'
```

### System configuration
**Rails version**: 5.0.1

**Ruby version**: ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]

**Comment by rafaelfranca** [comment]
Documentation is wrong. Can you send a patch?

**Comment by kiliancs** [comment]
@rafael thanks for the quick response.

Sure. I should be able to create a PR in the next couple days.

**Comment by kiliancs** [comment]
@rafaelfranca apparently the repository has already been corrected as per
https://github.com/rails/rails/commit/31e341a651e6863ce6bc34e4e2c3431c4f824544, but it's just not live at http://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-require.

I suppose this issue may be closed unless some other action has to be taken.

**Comment by rafaelfranca** [comment]
Should be fixed by 743bc0697d
